index.js

```diff
@@ -0,0 +1,24 @@
+"use strict"
+
+// The list of rules that can be used in Solhint can be found at:
+// https://protofire.github.io/solhint/docs/rules.html
+
+// The rules below uses the recommended rules from solhint with some
+// modifications
+module.exports = {
+  "extends": "solhint:recommended",
+  'rules': {
+    "imports-on-top": "error",                     // recommended uses warn
+    "use-forbidden-name": "error",                 // recommended uses warn
+    "no-unused-vars": "error",                     // recommended uses warn
+    "reason-string": ["error", {"maxLength":250}], // recommended uses warn
+    "max-line-length": ["error", 145],             // not present in recommended
+    "constructor-syntax": "error",                 // recommended uses warn
+    "visibility-modifier-order": "error",          // recommended uses warn
+    "ordering": "error",                           // not used in recommended
+    "var-name-mixedcase": "off",
+    "contract-name-camelcase": "off",
+    "const-name-snakecase": "off",
+    "compiler-version": "off"
```

Comment:
I do not see this one in https://github.com/keep-network/solium-config-keep/blob/master/index.js. Why do we add it?

Reply:
The `solium` version didn't check for the compiler version at all, that's why I turned it off (I wanted to get as close as possible to the original version).

We can enable this compiler version check, but we need to choose some value ([default version used by solhint is 0.5.8](https://github.com/protofire/solhint/blob/master/docs/rules/security/compiler-version.md)).

Reply:
Cool, your reasoning makes sense, let's leave it as-is. Especially that we have projects ranging from 0.5 to 0.8.

Reply:
Ok, I left the `compiler-version` check `off`ed